src/platform/plugins/shared/dashboard/server/types.ts

```diff
@@ -7,7 +7,47 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
+import { ContentManagementServerSetup } from '@kbn/content-management-plugin/server';
+
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface DashboardPluginSetup {}
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface DashboardPluginStart {}
+export interface DashboardPluginStart {
+  /**
+   * The return value of the contentClient.getForRequest method is a scoped client that can be used to perform CRUD and
```

Comment:
How about "Use contentClient.getForRequest to get a scoped client to perform CRUD and search operations for dashboards..."